app/lib/actions.ts

```diff
@@ -0,0 +1,48 @@
+'use server'
+import { z } from 'zod'
+import { Client, LogLevel } from '@notionhq/client'
+import { type QueryDatabaseResponse } from '@notionhq/client/build/src/api-endpoints'
+import { revalidatePath } from 'next/cache'
+import { redirect } from 'next/navigation'
+
+const CreateInvCodeSchema = z.object({
+  Code: z.string().min(3).max(4),
+})
+
+//TODO: Create a Service
+const INTEGRATION_SECRET = 'secret_W4UG7cmNojXJa6zMgLN34HaVmJGaimRxy2frviQws9r'
+const DATABASE_ID = '2c0b5c598a934d5b8f5265cfa6b31790'
```

Comment:
Should we save these constants in a different file or something, for avoiding to share these credentials? @CesarSernaV @i-mora

Reply:
If you look at line 12, there is a comment to pass that code to a service in the future, and pass those vars to an appSettings file in that case, since that code is duplicated.

Reply:
Oky, sounds good, thanks for your answer